app/controllers/device_stories_controller.rb

```diff
@@ -25,6 +25,11 @@ def show
     respond_with @device_story
   end
 
+  def show_airnote
+    @device_story = DeviceStory.where(device_urn: params[:device_urn]).first
+    respond_with @device_story
```

Comment:
Using `#find_by!` might be better in case when `param[:device_urn]` is invalid. And, it raises exception.

```suggestion
    respond_with DeviceStory.find_by!(device_urn: params[:device_urn])
    respond_with @device_story
  rescue ActiveRecord::RecordNotFound
    head :not_found
  end
```